windows/utils/win_install_updates.yml

```diff
@@ -32,25 +41,27 @@
     - name: "Install Windows Updates"
       ansible.windows.win_updates:
         server_selection: "windows_update"
-        category_names: '*'
-        log_path: "{{ win_udpates_log_file }}"
+        category_names: "{{ win_updates_category_names | default('*') }}"
+        reject_list: "{{ win_updates_reject_list | default('') }}"
+        log_path: "{{ win_updates_log_file }}"
         skip_optional: true
         state: "installed"
         reboot: true
-        reboot_timeout: 1800
+        reboot_timeout: 2700
       delegate_to: "{{ vm_guest_ip }}"
       register: win_updates_install_result
+      ignore_errors: "{{ win_updates_ignore_errors | default(false) }}"
```

Comment:
You can set `ignore_errors: true` at here, but add a when for below assert task:
```
when: win_updates_ignore_errors is undefined || not win_updates_ignore_errors
```
Therefor, the assert can be used when win_updates_ignore_errors is false.

Reply:
Will use var win_online_updates_succeed to trigger assert when not ignore error. Thanks.